src/dune_rules/gen_rules.ml

```diff
@@ -222,26 +225,12 @@ let gen_rules sctx dir_contents cctxs expander
     For_stanza.of_stanzas stanzas ~cctxs ~sctx ~src_dir ~ctx_dir ~scope
       ~dir_contents ~expander ~files_to_install
   in
-  let allow_approx_merlin =
```

Comment:
Does this mean that this PR makes `allow_approx_merlin` a no op?

Reply:
Yes. I will add a warning saying that this option is a deprecated no-op.